src/i18n/i18n.service.ts

```diff
@@ -107,9 +113,14 @@ export class Overridable {
 	 */
 	override(value: string | Observable<string>) {
 		this.isOverridden = true;
+		// To ensure that there are not multiple subscriptions created for the same observable, we
+		// unsubscribe if a subscription already exists for an observable before creating a new one.
+		if (this.subscription) {
```

Comment:
Shouldn't this go inside of `if (isObservable(value)) {` block so we only unsubscribe if we'll subscribe after?

Reply:
What happens if we have a subscription but someone passes a raw value in (for whatever reason)?

Reply:
Yup, makes sense.